src/Customer.js

```diff
@@ -1,3 +1,69 @@
-class Customer {}
+class Customer {
+  constructor(customerData, bookingData) {
+    this.id = customerData.id;
+    this.name = customerData.name;
+    this.bookings = [];
+
+    this.getBookings(bookingData);
+  }
+
+  getBookings(bookingData) {
+    const matchedBookings = bookingData.filter(booking => booking.userID === this.id);
+    this.bookings = matchedBookings;
+  }
+
+  sortBookings() {
+    return this.bookings.sort((a, b) => new Date(a.date) - new Date(b.date));
+  }
+
+  getPastBookings(inputDate) {
+    const results = this.bookings.filter(booking => {
```

Comment:
oh look... functionality for it...

Reply:
My functions for the past/future bookings are really similar... If you have any suggestions for combining them I would love to hear! Or maybe a good opportunity for future refactoring.